sdk/tx-kernel/src/lib.rs

```diff
@@ -0,0 +1,94 @@
+#![no_std]
+
+#[panic_handler]
+fn my_panic(_info: &core::panic::PanicInfo) -> ! {
+    loop {}
+}
+
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+extern crate alloc;
+
+use alloc::vec::Vec;
+// use miden_sdk_types::Felt;
+
+#[repr(transparent)]
+#[derive(Copy, Clone)]
+pub struct AccountId(Felt);
+
+impl From<AccountId> for Felt {
+    fn from(account_id: AccountId) -> Felt {
+        account_id.0
+    }
+}
+
+#[link(wasm_import_module = "miden:tx_kernel/account")]
+extern "C" {
+    #[link_name = "get_id<0x0000000000000000000000000000000000000000000000000000000000000000>"]
+    fn extern_account_get_id() -> AccountId;
+    #[link_name = "add_asset<0x0000000000000000000000000000000000000000000000000000000000000000>"]
+    fn extern_account_add_asset(_: Felt, _: Felt, _: Felt, _: Felt, _: i32);
+}
+
+#[link(wasm_import_module = "miden:tx_kernel/note")]
+extern "C" {
+    #[link_name = "get_inputs<0x0000000000000000000000000000000000000000000000000000000000000000>"]
+    fn extern_note_get_inputs(ptr: i32) -> i32;
+}
+
+#[inline(always)]
+pub fn get_id() -> AccountId {
+    unsafe { extern_account_get_id() }
+}
+
+// Temporary use u64 instead of Felt until https://github.com/0xPolygonMiden/compiler/issues/118#issuecomment-1978388977 is resolved
+
+const MAX_INPUTS: usize = 256;
+
+#[inline(always)]
+pub fn get_inputs() -> Vec<u64> {
```

Comment:
The `Felt` type is defined as i32 below, but then we're returning `Vec<u64>` here, shouldn't this be `Vec<Felt>`?

Reply:
Yes. For some weird reason, I temporarily switched `Felt` to be `i32`(instead of `i64`) until proper felts are implemented in the next PR, but then forgot to bring it back. Fixed in https://github.com/0xPolygonMiden/compiler/pull/140/commits/7b21de4a54a1f79834946685a9cf8a95254473c0